overlord/snapshotstate/snapshotmgr.go

```diff
@@ -46,27 +48,90 @@ var (
 	backendCheck         = (*backend.Reader).Check
 	backendRevert        = (*backend.RestoreState).Revert // ditto
 	backendCleanup       = (*backend.RestoreState).Cleanup
+
+	snapshotExpirationLoopInterval = time.Hour * 24 // interval between forgetExpiredSnapshots runs as part of Ensure()
```

Comment:
we are inside snapshots,   just autoExpirationInterval  ?

Reply:
Done